library/src/main/java/com/midtrans/httpclient/APIHttpClient.java

```diff
@@ -42,8 +42,8 @@ public APIHttpClient(Config config) {
 
         RequestBody body = null;
         if (!method.equals(GET)) {
-            MediaType mediaType = MediaType.parse("application/json");
-            body = RequestBody.create(mediaType, jsonParam.toString());
+            //MediaType mediaType = MediaType.parse("application/json");
+            body = RequestBody.create(null, jsonParam.toString());
```

Comment:
using `null` here will not remove `application/json` from content type header right? if so, then we are fine.

Reply:
Yes, should be no, we already put manually on headers to[ on this line](https://github.com/Midtrans/midtrans-java/blob/master/library/src/main/java/com/midtrans/httpclient/APIHttpClient.java#L190)